_core/smart/cli-wallet-quickstart.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: smart
+description: "CLI Wallet Guide"
+permalink: /:collection/:path.html
+---
+# Blockstack CLI Wallet Quickstart (Testnet)
+
+This quickstart guide will show you how to setup a Stacks wallet using the Blockstack JavaScript CLI and use it to send/receive Stacks tokens on Testnet. This wallet is intended for developer experimentation with Testnet only, do not use this wallet to store your tokens.
+
+## Prerequisites
+
+You will need to have `npm` installed.
+
+## Installation
+
+To install the Blockstack CLI, run the following command in terminal. 
+
+`npm install -g blockstack-cli` 
+
+## Creating a Wallet
+
+First, we are going to generate a new wallet for Testnet. To generate a wallet use the `make_keychain` command with the `-t` option for Testnet.
+
+```
+$ blockstack-cli make_keychain -t
+
+{
+  "mnemonic": "private unhappy random runway boil scissors remove harvest fatigue inherit inquiry still before mountain pet tail mad accuse second milk client rebuild salt chase",
+  "keyInfo": {
+    "privateKey": "381314da39a45f43f45ffd33b5d8767d1a38db0da71fea50ed9508e048765cf301",
+    "address": "ST1BG7MHW2R524WMF7X8PGG3V45ZN040EB9EW0GQJ",
+    "index": 0
+  }
+}
+```
+
+The mnemonic is your 24 word seed phrase which you should back up securely if you want access to this wallet again in the future. Once lost, it cannot be recovered. 
+
+The Stacks address associated with this wallet is: 
+`ST1BG7MHW2R524WMF7X8PGG3V45ZN040EB9EW0GQJ`
+
+Note that this is a Testnet address for use with Testnet only. You will use this address to receive tokens from the Testnet faucet or other people.
+
+The private key is what you will need to send tokens later. 
+
+## Getting Tokens
+
+To receive Testnet Stacks tokens, visit the Testnet stacks token [faucet](https://www.blockstack.org/faucet). Enter the Stacks address you generated above into the address field on the faucet page. 
```

Comment:
is this the confirmed final URL?

Reply:
No but we can quickly update this once the faucet URL is finalized.